patches/react-devtools-core+4.28.5.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/node_modules/react-devtools-core/dist/backend.js b/node_modules/react-devtools-core/dist/backend.js
+index d5837ee..1530af4 100644
+--- a/node_modules/react-devtools-core/dist/backend.js
++++ b/node_modules/react-devtools-core/dist/backend.js
+@@ -4671,7 +4671,7 @@ var SHOW_DURATION = 2000;
+ var timeoutID = null;
+ var overlay = null;
+ function hideOverlay(agent) {
+-  if (window.document == null) {
++  if (window.document == null || window.document.__isJsdom) {
```

Comment:
Additional `window.document.__isJsdom` references were added to this recreated patch in hopes of improving the compatibility with the dev tools. E.g., possibly enabling features like highlighting UI when hovering an element in the dev tools.